hicommonwealth/commonwealth
# Update Permissions Selection UI + Add UPDATE_POLL

## Link to Issue
Closes: #10009

## Description of Changes
- Redesigned the Topic Permissions UI to allow:
   - Custom combinations of permissions.
   - Scalable additions for new permissions.
- Added Voting on Polls (UPDATE_POLL) as a new permission option. 

## "How We Fixed It"

## Test Plan
- Navigate to community page.
- Click on "Members & Groups" on the Side Bar.
- Click on "Create Group".
- Navigate to Gated Topics Section.
- Add multiple topics and topic permission then Save.
- Verify that the permission are applied to the group of the respective topics.
- Also verify the edit group behaviour by repeating the above steps.

## Deployment Plan

1. 

## Other Considerations

-
